tests/scripts/kubeadm.sh

```diff
@@ -30,12 +29,48 @@ install() {
     done
 
     if [ "${kube_ready}" == "False" ]; then
-        echo "k8s node never went to Ready status"
+        echo "k8s master node never went to Ready status"
         exit 1
     fi
 
-    echo "k8s node in Ready status"
+    echo "k8s master node in Ready status"
+}
+install_node(){
+    echo "inside install node function"
+    echo "kubeadm join ${1} ${2} ${3} --skip-preflight-checks"
+    sudo kubeadm join ${1} ${2} ${3} --skip-preflight-checks || true
```

Comment:
Looks like this requires 3 arguments: it should check for that and fail if there are more or less than 3 arguments, or the caller should ensure that it is only called with 3 arguments.